### PR TITLE
[FIX] Correctly aligned input element of custom user status component

### LINKED ISSUE #16145: Input element of "user-status-custom" component is not alligned correctly.
### Description:
Input element of "user-status-custom" component does not have enough margin and is very close to the text "Presence"

### Steps to reproduce:

1. Go to Administration
2. Click "Custom User Status"
3. Click on add button

### Expected behavior:
There should be enough margin/padding

### Actual behavior:
It is not so.

imageedit_4_3261222051

###### Additional context
I am working on it.

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Closes #16145 

## Description:
Used default value of margin used in different components to keep UI consistent.

### Before fix:
imageedit_4_3261222051
### After fix:
aaaa
bbb

### CODE REVIEW

File: app/user-status/client/admin/userStatusEdit.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.rc-select {
+    margin: .5rem 0;
+}
```
Comment (sampaiodiego) [inline app/user-status/client/admin/userStatusEdit.css]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4776501)

File: app/user-status/client/admin/userStatusEdit.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.rc-select {
+    margin: .5rem 0;
```
Comment (sampaiodiego) [inline app/user-status/client/admin/userStatusEdit.css]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4776501)

Comment (sampaiodiego) [inline app/user-status/client/admin/userStatusEdit.css]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4776501)

**Comment by ashwaniYDV** [comment]
@sampaiodiego I have fixed the stylelint issue.

**Comment by ashwaniYDV** [comment]
@gabriellsh I have made the requested changes.
Thank you 😇

**Pull request merged**